src/reducers/secrets.js

```diff
@@ -80,9 +80,10 @@ function isFetching(state = false, action) {
 function errorMessage(state = null, action) {
   switch (action.type) {
     case 'SECRETS_FETCH_FAILURE':
-    case 'SECRET_DELETE_FAILURE':
     case 'SECRET_CREATE_FAILURE':
       return action.error;
+    case 'SECRET_DELETE_FAILURE':
```

Comment:
same as the cases above, change can be reverted

Reply:
I'm not sure I understand, this case is separate to the ```SECRET_CREATE_FAILURE``` case?

Reply:
you're doing the same thing here (`return action.error;`) as in the cases above so there's no functional difference from what was there before, it's just duplicating the return unnecessarily

Reply:
ok, reverted